### PR TITLE
[excel] (Samples) Add buttons to workbook downloads

### PR DESCRIPTION
**Pull request by AlexJerabek**
I found a download link I knew to exist in the PivotTable article hard to find. This PR turns most of the sample workbook download links into buttons.
(Note that the links to zip files couldn't be changed. OPS has weird limitations around zip files that I'm investigating separately.)

### CODE REVIEW

File: docs/resources/samples/add-excel-comments.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ This sample shows how to add comments to a cell including [@mentioning](https://
 
 ## Sample Excel file
 
-Download [add-excel-comments.xlsx](add-excel-comments.xlsx) for a ready-to-use workbook. Add the following script to try the sample yourself!
+> [!div class="nextstepaction"]
+> [Download the sample workbook](add-excel-comments.xlsx)
```
Comment (alison-mk) [inline docs/resources/samples/add-excel-comments.md:27]: I have one mild concern for this change and the ones like it. This concern isn't relevant for the articles that include the button in a series of numbered steps.

I view this new button as a suggestion or option, like the reader can choose to click the button to get the sample workbook, or they can just continue onward without downloading the sample workbook. I think the previous version made it more clear that the sample workbook is a requirement, that it must be downloaded to successfully complete the samples. 

This concern is most relevant to a first time Office Scripts sample reader—repeat readers will know that the sample workbook must be downloaded.

Here are a couple ideas for you to consider: 

One potential solution could be updating the following text to directly mention the sample workbook: "Add the following script to the sample workbook try the sample yourself!"

Another might be updating the h2 to clarify that the sample workbook is a requirement part of the setup, perhaps: "Setup: Sample Excel file"

**Review by alison-mk — APPROVED** [review]
I think adding a button here is an improvement! I'm approving, and I left a comment with some additional thoughts.

File: docs/resources/samples/add-excel-comments.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/resources/samples/add-excel-comments.md:27]: Thanks for thinking through this. I'll revisit the samples and take your suggestions to make sure it's clear the download is required (and just a harmless workbook).

Comment (AlexJerabek) [reply docs/resources/samples/add-excel-comments.md:27]: @alison-mk,
How about a hybrid of both your suggestions?

```
## Setup: Sample Excel file

This workbook contains the data, objects, and formatting expected by the script.

> [!div class="nextstepaction"]
> [Download the sample workbook](add-excel-comments.xlsx)

## Sample code: Add comments

Add the following script to the sample workbook and try the sample yourself!
```

Comment (alison-mk) [reply docs/resources/samples/add-excel-comments.md:27]: I like that! Very clear. Looks great to me.

**Comment by AlexJerabek** [comment]
@alison-mk Updated based on your suggestions.

**Review by alison-mk — APPROVED** [review]

**Pull request merged**